pkg/providers/aws/provider_postgres.go

```diff
@@ -158,6 +158,8 @@ func (p *PostgresProvider) createRDSInstance(ctx context.Context, cr *v1alpha1.P
 		errMsg := "error reconciling rds networking"
 		return nil, croType.StatusMessage(errMsg), errorUtil.Wrap(err, errMsg)
 	}
+	errMsg := "we don;t wanna create other stufff and thngs"
```

Comment:
This will be removed later. It will return for now so the logic only tries to create the vpc.